<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IFOCUSODC-PC47" name="API_VootKids.ResetPin" tests="1" failures="1" timestamp="23 Apr 2018 11:32:38 GMT" time="14.486" errors="0">
  <testcase name="reset_Pin" time="14.486" classname="API_VootKids.ResetPin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [This custom field is invalid, please use a correct or valid custom field.] but found [Pin reset was successful.],
	expected [This custom field is invalid, please use a correct or valid custom field.] but found [Pin reset was successful.],
	expected [This custom field is invalid, please use a correct or valid custom field.] but found [The new pin must not same as old pin],
	expected [This custom field is invalid, please use a correct or valid custom field.] but found [invalid old pin]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [This custom field is invalid, please use a correct or valid custom field.] but found [Pin reset was successful.],
	expected [This custom field is invalid, please use a correct or valid custom field.] but found [Pin reset was successful.],
	expected [This custom field is invalid, please use a correct or valid custom field.] but found [The new pin must not same as old pin],
	expected [This custom field is invalid, please use a correct or valid custom field.] but found [invalid old pin]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at API_VootKids.ResetPin.reset_Pin(ResetPin.java:133)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- reset_Pin -->
</testsuite> <!-- API_VootKids.ResetPin -->
